Keas.Mvc/Services/EventService.cs

```diff
@@ -24,6 +32,57 @@ public EventService(IHistoryService historyService, INotificationService notific
             var history = await _historyService.KeyCreated(key, user);
             await _notificationService.KeyCreatedUpdatedInactive(key, history);
         }
+
+        public async Task TrackAssignKey(Key key, User user)
+        {
+            var history = await _historyService.KeyAssigned(key, user);
+            await _notificationService.KeyAssigned(key, history);
+
+        }
+
+        public async Task TrackUnAssignKey(Key key, User user)
+        {
+            var history = await _historyService.KeyUnassigned(key, user);
+            await _notificationService.KeyUnAssigned(key, history);
+        }
+
+        public async Task TrackCreateEquipment(Equipment equipment, User user)
+        {
+            var history = await _historyService.EquipmentCreated(equipment, user);
+            await _notificationService.EquipmentCreatedUpdatedInactive(equipment, history);
+        }
+
+        public async Task TrackAssignEquipment(Equipment equipment, User user)
+        {
+            var history = await _historyService.EquipmentAssigned(equipment, user);
+            await _notificationService.EquipmentAssigned(equipment, history);
+        }
+
+        public async Task TrackUnAssignEquipment(Equipment equipment, User user)
+        {
+            var history = await _historyService.EquipmentUnassigned(equipment, user);
+            await _notificationService.EquipmentUnAssigned(equipment, history);
+        }
+
+        public async Task TrackCreateAccess(Access access, User user)
+        {
+            var history = await _historyService.AccessCreated(access, user);
+            await _notificationService.AccessCreatedUpdatedInactive(access, history);
+        }
+
+        public async Task TrackAssignAccess(AccessAssignment accessAssignment, User user)
+        {
+            var history = await _historyService.AccessAssigned(accessAssignment, user);
+            await _notificationService.AccessAssigned(accessAssignment, history);
+
+        }
+
+        public async Task TrackUnAssignAccess(AccessAssignment accessAssignment, User user)
```

Comment:
@jcubbage since the initiator is always the currently logged in user, shouldn't we just grab via the security service here?  Or optionally maybe just grab if passed in user == null?